apps/ukvi-complaints/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable consistent-return */
```

Comment:
did you need to add this because you are throwing an error?

Reply:
This was for a file that came from Marvel and we didn't do any work on. Its because the arrow functions used don't have a return for every route within them. Don't want to mess too much with the code as it's not to do with the integration but was throwing up linting errors.

Reply:
ok fair enough